expr/ztests/pick.yaml

```diff
@@ -0,0 +1,20 @@
+zql: pick s,x
```

Comment:
This is the only mention of pick within package expr, so expr/ztests/ feels like an odd place for it, but I don't have any good ideas about where else it should go.

Reply:
It's entirely implemented in expr/cutter.go

Reply:
You and I know that, but but someone new to the code won't. 😉 I'll add a comment later.